frontend/packages/dev-console/integration-tests/support/pageObjects/operators-po.ts

```diff
@@ -1,7 +1,7 @@
 export const operatorsPO = {
   search: '[data-test="search-operatorhub"] input[aria-label="Filter by keyword..."]',
   nav: {
-    operators: '[data-quickstart-id="qs-nav-operators"]',
+    ecosystem: '[data-quickstart-id="qs-nav-ecosystem"]',
```

Comment:
Looking for quickstart-id, which has not been added for the Ecosystem nav item.

Reply:
As the Operator nav item is being removed from the navigation in this PR. Please look and update all the e2e tests  which are using it.